yarn-project/end-to-end/src/guides/up_quick_start.sh

```diff
@@ -4,8 +4,8 @@
 set -eux
 
 # docs:start:declare-accounts
-ALICE="0x25048e8c1b7dea68053d597ac2d920637c99523651edfb123d0632da785970d0"
-BOB="0x115f123bbc6cc6af9890055821cfba23a7c4e8832377a32ccb719a1ba3a86483"
+ALICE="0x0f394e8bd156e15153376a711e3054821c2a1c1047dcfb3745d636a57fb42ab1"
+BOB="0x2b67f90f0044596190644ddafea4152de47bd4781559493860fa7358e19d090a"
```

Comment:
The address is reliant on a pedersen hash and since the hash changed, the address, contract address and secret hash all changed with it